CONTRIBUTING.md

```diff
@@ -0,0 +1,153 @@
+# Development Ideology
+
+Truths which we believe to be self-evident (adapted from [TextSecure's](https://github.com/WhisperSystems/TextSecure/blob/master/contributing.md))
+
+1. **The answer is not more options.** If you feel compelled to add a preference that's exposed to the user, it's very possible you've made a wrong turn somewhere.
+2. **There are no power users.** The idea that some users "understand" concepts better than others has proven to be, for the most part, false. If anything, "power users" are more dangerous than the rest, and we should avoid exposing dangerous functionality to them.
+3. **If it's "like PGP," it's wrong.** PGP is our guide for what not to do.
```

Comment:
They make sense to me in the context of Jenkins plugins. I removed the one that I felt did not make sense. Care to point out ones you feel could be rephrased to make more obvious?

Sent from my iPhone

> On 5 Aug 2015, at 22:19, Daniel Beck <notifications@github.com> wrote:
> 
> In CONTRIBUTING.md:
> 
> > @@ -0,0 +1,153 @@
> > +# Development Ideology
> > +
> > +Truths which we believe to be self-evident (adapted from [TextSecure's](https://github.com/WhisperSystems/TextSecure/blob/master/contributing.md))
> > +
> > +1. **The answer is not more options.** If you feel compelled to add a preference that's exposed to the user, it's very possible you've made a wrong turn somewhere.
> > +2. **There are no power users.** The idea that some users "understand" concepts better than others has proven to be, for the most part, false. If anything, "power users" are more dangerous than the rest, and we should avoid exposing dangerous functionality to them.
> > +3. **If it's "like PGP," it's wrong.** PGP is our guide for what not to do.
> Half of these don't even make sense for Jenkins plugins. I mean, sure, they're right, but this isn't TextSecure, so they shouldn't be front and center.
> 
> —
> Reply to this email directly or view it on GitHub.
>